terraspace/app/stacks/peer/locals.tf

```diff
@@ -0,0 +1,348 @@
+      protocol    = "udp"
+      from_port   = 9094
+      to_port     = 9094
+      type        = "egress"
+      self        = true
+    }
+
+    node_to_node_443_ing = {
+      description = "Node to node HTTPS"
+      protocol    = "tcp"
+      from_port   = 443
+      to_port     = 443
+      type        = "ingress"
+      self        = true
+    }
+
+    node_to_node_443_out = {
+      description = "Node to node HTTPS"
+      protocol    = "tcp"
+      from_port   = 443
+      to_port     = 443
+      type        = "egress"
+      self        = true
+    }
+
+    node_to_node_80_ing = {
+      description = "Node to node HTTP"
+      protocol    = "tcp"
+      from_port   = 80
+      to_port     = 80
+      type        = "ingress"
+      self        = true
+    }
+
+    node_to_node_80_out = {
+      description = "Node to node HTTP"
+      protocol    = "tcp"
+      from_port   = 80
+      to_port     = 80
+      type        = "egress"
+      self        = true
+    }
+
+    argocd_repo_server_8081_ing = {
+      description = "Node to node argocd-repo-server"
+      protocol    = "tcp"
+      from_port   = 8081
+      to_port     = 8081
+      type        = "ingress"
+      self        = true
+    }
+    argocd_repo_server_8081_eg = {
+      description = "Node to node argocd-repo-server"
+      protocol    = "tcp"
+      from_port   = 8081
+      to_port     = 8081
+      type        = "egress"
+      self        = true
+    }
+
+    argocd-application-controller_8082_ing = {
+      description = "Node to node argocd-application-controller"
+      protocol    = "tcp"
+      from_port   = 8082
+      to_port     = 8082
+      type        = "ingress"
+      self        = true
+    }
+    argocd-application-controller_8082_eg = {
+      description = "Node to node argocd-application-controller"
+      protocol    = "tcp"
+      from_port   = 8082
+      to_port     = 8082
+      type        = "egress"
+      self        = true
+    }
+
+
+    argocd_redis_6379_ing = {
+      description = "Node to node argocd-redis"
+      protocol    = "tcp"
+      from_port   = 6379
+      to_port     = 6379
+      type        = "ingress"
+      self        = true
+    }
+    argocd_redis_6379_eg = {
+      description = "Node to node argocd-redis"
+      protocol    = "tcp"
+      from_port   = 6379
+      to_port     = 6379
+      type        = "egress"
+      self        = true
+    }
+
+    traceroute_33434_33464_eg = {
+      description = "Enable troubleshoot: UDP Traceroute to all internet"
+      protocol    = "UDP"
+      from_port   = 33434
+      to_port     = 33655
+      type        = "egress"
+      cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    traceroute_ICMP_eg = {
+      description = "Enable troubleshoot: ICMP Traceroute to all internet"
+      protocol    = "ICMP"
+      from_port   = -1
+      to_port     = -1
+      type        = "egress"
+      cidr_blocks = ["0.0.0.0/0"]
+    }
```

Comment:
do we need these trouble shooting rules anymore?

Reply:
In that case, I'd have them commented out with an informational message to uncomment if you'd like to enable tracerouting.  You could make it "smarter" by having a `var.enable_tracerouting` to include/exclude those rules.  Given it is just for break-glass troubleshooting, I think thats overkill.

Reply:
Ended up doing the enable_tracerouting variable! :)
It might be overkill but at least give us the flexibility of changing without affecting other environments